### PR TITLE
#165 Okr Component Page

### PR DESCRIPTION
**Pull request by deimonbody**
Added :
- Store of okr
- Fixed some endpoins
- Abilities:
  -delete okr
  -delete objective
  -delete key-result
  -edit key result
  -edit objective
  -close okr

### CODE REVIEW

File: frontend/src/validation-schemas/okr/okr.validation-schema.ts [hunk 1]
```diff
@@ -48,4 +49,22 @@ const okrValidationSchema = (isEdit: boolean): Joi.ObjectSchema<IOkr> => {
   });
 };
 
-export { okrValidationSchema };
+const objectiveValidationSchema = (): Joi.ObjectSchema<ObjectiveValues> => {
+  //Messages for the objective schema
```
Comment (nikita-remeslov) [inline frontend/src/validation-schemas/okr/okr.validation-schema.ts]: ?

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/validation-schemas/okr/okr.validation-schema.ts (hunk 1 above)
Comment (nikita-remeslov) [reply frontend/src/validation-schemas/okr/okr.validation-schema.ts]: Do you need this comments?

File: frontend/src/components/okr/styles.scss [hunk 2]
```diff
@@ -14,6 +14,13 @@
     'time time time';
 }
 
+.okr-disable {
+  background-color: #8080801f !important;
```
Comment (nikita-remeslov) [inline frontend/src/components/okr/styles.scss]: Do you really need `!important`

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

**Review by aleksandr-kuharenko — APPROVED** [review]

**Review by nikita-remeslov — APPROVED** [review]

**Comment by nikita-remeslov** [comment]
Conflicts

**Review by aleksandr-kuharenko — APPROVED** [review]

**Pull request merged**